examples/quantization_aware_training/torch/anomalib/main.py

```diff
@@ -0,0 +1,161 @@
+    def transform_fn(data_item):
+        return data_item["image"]
+
+    test_loader = datamodule.test_dataloader()
+    calibration_dataset = nncf.Dataset(test_loader, transform_fn)
+
+    # Quantize the inference model using Post-Training Quantization
+    inference_model = model.model
+    quantized_inference_model = nncf.quantize(model=inference_model, calibration_dataset=calibration_dataset)
+
+    # Deepcopy the original model and set the quantized inference model
+    quantized_model = deepcopy(model)
+    quantized_model.model = quantized_inference_model
+
+    # Create engine for the quantized model
+    engine = Engine(task=TaskType.SEGMENTATION, default_root_dir=INT8_RESULTS_ROOT, max_epochs=1)
+
+    # Validate the quantized model
+    print("Test results for INT8 model after PTQ:")
+    int8_init_test_results = engine.test(model=quantized_model, datamodule=datamodule)
+
+    ###############################################################################
+    # Step 3: Fine tune the quantized model
+    print(os.linesep + "[Step 3] Fine tune the quantized model")
+
+    engine.fit(model=quantized_model, datamodule=datamodule)
+    print("Test results for INT8 model after QAT:")
+    int8_test_results = engine.test(model=quantized_model, datamodule=datamodule)
+
+    ###############################################################################
+    # Step 4: Export models
+    print(os.linesep + "[Step 4] Export models")
+
+    # Export FP32 model to OpenVINO™ IR
+    fp32_ir_path = engine.export(model=model, export_type=ExportType.OPENVINO, export_root=FP32_RESULTS_ROOT)
+    print(f"Original model path: {fp32_ir_path}")
+    fp32_size = get_model_size(fp32_ir_path)
+
+    # Export INT8 model to OpenVINO™ IR
+    int8_ir_path = engine.export(model=quantized_model, export_type=ExportType.OPENVINO, export_root=INT8_RESULTS_ROOT)
+    print(f"Quantized model path: {int8_ir_path}")
+    int8_size = get_model_size(int8_ir_path)
+
+    ###############################################################################
+    # Step 5: Run benchmarks
+    print(os.linesep + "[Step 5] Run benchmarks")
+
+    print("Run benchmark for FP32 model (IR)...")
+    fp32_fps = run_benchmark(fp32_ir_path, shape=[1, 3, 256, 256])
+
+    print("Run benchmark for INT8 model (IR)...")
+    int8_fps = run_benchmark(int8_ir_path, shape=[1, 3, 256, 256])
+
+    ###############################################################################
+    # Step 6: Summary
+    print(os.linesep + "[Step 6] Summary")
+
+    fp32_f1score = fp32_test_results[0]["image_F1Score"]
+    int8_init_f1score = int8_init_test_results[0]["image_F1Score"]
+    int8_f1score = int8_test_results[0]["image_F1Score"]
+
+    print(f"Accuracy drop after PTQ:                {fp32_f1score - int8_init_f1score:.3f}")
+    print(f"Accuracy drop after QAT:                {fp32_f1score - int8_f1score:.3f}")
+    print(f"Model compression rate:                 {fp32_size / int8_size:.3f}")
```

Comment:
What do you think about a summary table as it does in the resnet18 qat example?

Reply:
Printing the table takes a lot of lines, I prefer a simple solution.